configurations/pipeline_configuration.py

```diff
@@ -43,15 +43,37 @@
         )
     ],
     csv_sources=[
+        # Hormud
         CSVSource(
             "gs://avf-project-datasets/2022/RVI-ELECTIONS/recovered_hormuud_2022_03_12_to_2022_03_18_de_identified.csv",
             engagement_db_datasets=[
                 # This contains data from 12th March, until 18th March.
-                CSVDatasetConfiguration("rvi_elections_s01e01", 
+                CSVDatasetConfiguration("rvi_elections_s01e01",
                                         start_date=isoparse("2022-03-12T00:00:00+03:00"), 
                                         end_date=isoparse("2022-03-19T00:00:00+03:00"))
             ],
             timezone="Africa/Mogadishu"
+        ),
+        CSVSource(
+            "gs://avf-project-datasets/2022/RVI-ELECTIONS/recovered_hormuud_2022_03_19_to_2022_03_25_de_identified.csv",
+            engagement_db_datasets=[
+                # This contains data from 19th March, until 25th March.
+                CSVDatasetConfiguration("rvi_elections_s01e02",
+                                        start_date=isoparse("2022-03-19T00:00:00+03:00"),
+                                        end_date=isoparse("2022-03-26T00:00:00+03:00"))
+            ],
+            timezone="Africa/Mogadishu"
+        ),
+        # Golis
+        CSVSource(
+            "gs://avf-project-datasets/2022/RVI-ELECTIONS/recovered_golis_2022_03_27_to_2022_03_28_de_identified.csv",
```

Comment:
Is this missing any data, or does it completely cover the 27th and 28th?

Reply:
It completely covers the 28th but on 27th there is no data in the afternoon.

Reply:
I've updated the recovered data